openedx/core/djangoapps/bookmarks/models.py

```diff
@@ -81,7 +81,7 @@ def create(cls, data):
 
             xblock_cache = XBlockCache.create({
                 'usage_key': usage_key,
-                'display_name': block.display_name,
+                'display_name': block.display_name or '',
```

Comment:
@cahrens  This is how the UI will look with empty default name. 

<img width="901" alt="screen shot 2016-01-07 at 7 17 40 pm" src="https://cloud.githubusercontent.com/assets/2994509/12173111/c4a9563c-b576-11e5-8ba8-7153a84de3fb.png">